example/python/tx-example.py

```diff
@@ -78,7 +78,7 @@
 
 if not query_response.HasField("asset_response"):
     print("Query response error")
```

Comment:
this is an example, it should be as descriptive as possible.
this print does not say anything, and this example lacks comments